polaris/dataset/converters/_zarr.py

```diff
@@ -27,7 +27,7 @@ class ZarrConverter(Converter):
     to a single column and each array contains the values for all datapoints in that column.
     """
 
-    def convert(self, path: str, factory: "DatasetFactory") -> FactoryProduct:
+    def convert(self, path: str, factory: "DatasetFactory", **kwargs) -> FactoryProduct:
```

Comment:
**Suggestion:** Similar comment as for PDB converter. I think appending is also useful functionality for the Zarr converter.